v0/destinations/splitio/data/GroupConfig.json

```diff
@@ -0,0 +1,17 @@
+[{
+    "destKey": "name",
+    "sourceKeys": [
+      "traits.name",
+      "context.traits.name"
```

Comment:
we should not take from context.traits. as it contains information about the user and not group.